templates/shared/combomarkdowneditor.tmpl

```diff
@@ -3,19 +3,22 @@ Template Attributes:
 * ContainerId: id attribute for the container element
 * ContainerClasses: additional classes for the container element
 * MarkdownPreviewUrl: preview url for the preview tab
+* MarkdownPreviewMode: the preview mode (default to comment)
 * MarkdownPreviewContext: preview context for the preview tab
 * TextareaName: name attribute for the textarea
 * TextareaContent: content for the textarea
 * TextareaPlaceholder: placeholder attribute for the textarea
 * TextareaAriaLabel: aria-label attribute for the textarea
+* TextareaMaxLength: maxlength attribute for the textarea
 * DropzoneParentContainer: container for file upload (leave it empty if no upload)
 * DisableAutosize: whether to disable automatic height resizing
+* HideRepoButtons: hide buttons that belongs to a repo
```

Comment:
I guess something like "MarkdownPreviewMode" already implies which buttons are needed?

Reply:
I don't know what the difference between `comment` and `markdown` in detail are. I only know that markdown doesn't render issue references, so I think it's the best to keep this a separate option. It doesn't hurt and it can maybe help to avoid potential problems in the future.